src/test/java/UserServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+import com.mateacademy.springmvc.entity.User;
+import com.mateacademy.springmvc.service.UserService;
+import lombok.AllArgsConstructor;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class UserServiceImplTest {
```

Comment:
Did you run it on production db?

Reply:
yes i run it

Reply:
It should not run on production db. You should use for tests h2 database or mock repository.